src/util/wren2cstring.c

```diff
@@ -34,27 +40,27 @@ char* WREN2CSTRING_readEntireFile(char* path, size_t* lengthPtr)
   return source;
 }
 
-int WREN2CSTRING_encodeAndDump(int argc, char* args[])
+int WREN2CSTRING_encodeAndDump(int argc, char* args[], int inDOME)
```

Comment:
As an alternative to this new parameter, you could just adjust the args pointer you pass in to the function to be one step along.